src/worker.ts

```diff
@@ -1,16 +1,18 @@
-import throng from "throng";
-import { start } from "./worker/main";
-import InitializeSentry from "./config/sentry";
+import throng from 'throng';
+import { initializeSentry } from './config/sentry';
+import { start } from './worker/main';
 
 const isProd = process.env.NODE_ENV === 'production';
-InitializeSentry();
+initializeSentry();
```

Comment:
This needs to be in the `worker/main.ts` start function or else it won't work with node clustering.